apps/web/src/pages/applicants.tsx

```diff
@@ -82,6 +77,27 @@ const Applicants = () => {
     }
   };
 
+  const changeRoute = (keyword: string, status: number) => {
```

Comment:
this'll produce a code smell since another 'status' is defined one line 30

Reply:
changed it too.